### ISSUE TITLE
Able to crash editor when editing tables quickly

### ISSUE THREAD

**Issue by will-newmarch** [froala/wysiwyg-editor#2169]
##### Expected behavior.
Able to edit table cells quickly

##### Actual behavior.
When trying to edit table cells quickly the editor eventually freezes

##### Steps to reproduce the problem.
See GIF, though essentially, add multiple tables inside each other and edit different cells in quick succession - GIF showing issue https://cl.ly/1y0G1d3m2w2r

##### OS.
Mac OSX

##### Browser.
Chrome Version 60.0.3112.101 (Official Build) (64-bit)

**Comment by dianaprajescu** [comment]
Thanks for reporting this. However we can't figure out the exact steps to reproduce the issue. Is it a specific action you are trying to achieve that causes the editor to freeze? We can't even say if it's a simple click, double click or click and drag. Also what is the origin of the click? Any extra feedback you can provide is highly appreciated.

**Comment by will-newmarch** [comment]
Hi Diana, yes of course, sorry I wasn't very detailed in my explanation. On the GIF I sent over I have created a table and then created a table inside one of the table cells and repeated this making another table in another table cell.

I have then single clicked into a cell and typed text, before single clicking into another cell and typing more text, and then repeating this in quick succession. This might seem like a bit of an edge-case though it seems we've been having people using the editor quite quickly like this and it seems to crash. I've only managed to get it to crash when focusing in and out of table cells and editing text in inside them very quickly. The more tables the easier it is to crash as well.

Let me know if you need any more information.

Will

**Comment by stefanneculai** [comment]
@will-newmarch Unfortunately, without having a consistent way of reproducing this it makes it impossible for us to fix it. While we've been able to reproduce it when typing and selecting chaotic, we couldn't find a logical way to reproduce it even when we recorded the screen and watched it in slow-motion. I am going to close this issue for now, and we would be happy to look again into it if there is a clear way to reproduce it.
